scripts/testnet/include/nodes.sh

```diff
@@ -123,12 +123,10 @@ assembleCommand_startObserverNode() {
   OBSERVER_INDEX=$2
   let "PORT = $PORT_ORIGIN_OBSERVER + $OBSERVER_INDEX"
   let "RESTAPIPORT=$PORT_ORIGIN_OBSERVER_REST + $OBSERVER_INDEX"
-  let "KEY_INDEX=$TOTAL_NODECOUNT - $OBSERVER_INDEX - 1"
   WORKING_DIR=$TESTNETDIR/node_working_dirs/observer$OBSERVER_INDEX
 
   local nodeCommand="./node \
         -port $PORT -log-save -log-level $LOGLEVEL -rest-api-interface localhost:$RESTAPIPORT \
-        -tx-sign-sk-index $KEY_INDEX -sk-index $KEY_INDEX \
```

Comment:
Should revert the `-sk-index $KEY_INDEX \` part as this is used for the block sign key

Reply:
done.